### ISSUE TITLE
[Feature request] Pipewire support

### ISSUE THREAD

**Issue by RannyBergamotte** [rbn42/panon#110]
Would be nice to have pipewire support.

Seems like a really cool widget though!!

**Comment by RannyBergamotte** [comment]
Closed cause I am stupid. Sorry about it! Didn't realize pipewire was already supported.

**Comment by MurzNN** [comment]
@RannyBergamotte Could you please describe how to configure work with PipeWire?
For me, I see only PulseAudio and PortAudio as the back-end.
With PulseAudio - no visualization, with PortAudio - I see only microphone input as the visualization, not speakers, and no ways to choose the right channel in settings.

**Comment by RannyBergamotte** [comment]
It was port audio that worked with pipewire I think, and yeah it selects the mic by default. So what I did, is using something like qpwgraph or some pipewire audio routing interface, I found the node for the panon input, and then connected my speaker monitor out to the panon input. I think there is also probably a way in the pipewire config files to set it up so that it auto connects to a specific thing, but I am too dum to figure that out, so I just use qpwgraph everytime. Hope that helps, lemme know if it goes well for you!
